libs/cluster/Session/RespClusterReplicationCommands.cs

```diff
@@ -12,28 +13,25 @@ internal sealed unsafe partial class ClusterSession : IClusterSession
         /// <summary>
         /// Implements CLUSTER REPLICAS command
         /// </summary>
-        /// <param name="count"></param>
         /// <param name="invalidParameters"></param>
         /// <returns></returns>
-        private bool NetworkClusterReplicas(int count, out bool invalidParameters)
+        private bool NetworkClusterReplicas(out bool invalidParameters)
         {
             invalidParameters = false;
 
             // Expecting exactly 0 arguments
-            if (count != 0)
+            if (parseState.Count != 0)
```

Comment:
bug: this should be != 1 (exactly one argument).
I know it is also in main, but if you could fix it that will be great!